src/components/Navigation.js

```diff
@@ -1,18 +1,19 @@
 import React from 'react';
+import {Link} from 'react-router-dom';
 
 const Navigation = () => {
   return (
     <div>
       <div className="App">
         <h1>React Router Mini</h1>
         <div>
-          <a href="">Home</a>
+          <a href="/">Home</a>
         </div>
         <div>
-          <a href="">About</a>
+          <a href="/about">About</a>
         </div>
         <div>
-          <a href="">Contact</a>
+          <a href="/contact">Contact</a>
```

Comment:
I don't see any Link tags - You'd want to wrap these a tags in Link Tags so we can route to specific components